docs/src/components/VisualList.tsx

```diff
@@ -69,10 +65,7 @@ const Visual = ({
   smallVisual,
   visualName,
 }: VisualProps) => {
-  const [copied, setCopied] = useState(false);
-  useEffect(() => {
-    copyTimeout(copied, setCopied);
-  }, [copied, setCopied]);
+  const [copied, setCopied] = useCopyToClipboard();
```

Comment:
I would rename `setCopied` to `copy` or `copyToClipboard` instead, to suggest a text argument rather than boolean. (And rename it elsewhere, too, not only here.)